lib/sensu-plugins-aws/cloudwatch-common.rb

```diff
@@ -2,8 +2,17 @@
 module CloudwatchCommon
   include Common
 
-  def client
-    Aws::CloudWatch::Client.new
+  def aws_config
```

Comment:
why is this needed? Fetching of creds should happen automatically by:
> `include Common`

https://github.com/sensu-plugins/sensu-plugins-aws/blob/10.0.3/lib/sensu-plugins-aws/common.rb#L25-L31

Reply:
I'm not a fan of the way common does it, all the nil checking isn't needed. But I can defer to that.

Reply:
If you think there is a better way to do it in common I am all for it, We just want to have a standard way of fetching aws credentials. Some checks supporting one method but not another leads to a poor user experience and maintenance nightmare.